helm/chart/router/Chart.yaml

```diff
@@ -21,4 +21,4 @@ version: 0.1.10
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "v0.9.4"
+appVersion: "v0.9.5"
```

Comment:
You have to increment `version` if you change `appVersion`.

Reply:
Fixed in [3521214](https://github.com/apollographql/router/pull/1276/commits/352121405002346caaf7944037440daccbd1ce0b), but please check.

Reply:
Trying again in [69da791](https://github.com/apollographql/router/pull/1276/commits/69da791eeba9b62eb3c14c98558f81ea199f02a1)